Red Steel Flask

Nwitlyck opened the pull request.

Nwitlyck committed:
Blue Steel Flask
-Added a blue steel flask variant
-Change finishUsingItem to have a condition that if item has a max durability of -1 it will not get damage

Nwitlyck committed:
change to red steel
-Change textures and all reference to be red steel and not blue steel

Gaelmare commented:
Looks pretty good! Please update and use the python resource gen for the
new type rather than hand generating the json.

On Sun, Mar 2, 2025, 10:29 Nwitlyck ***@***.***> wrote:

> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/Gaelmare/WaterFlasks/pull/30
> Commit Summary
>
>    - 974d361
>    <https://github.com/Gaelmare/WaterFlasks/pull/30/commits/974d361a465d0981f89907efe05755243ea26344>
>    Blue Steel Flask
>    - c33b968
>    <https://github.com/Gaelmare/WaterFlasks/pull/30/commits/c33b96803ba80cf6805ef904c728b59635d09ca4>
>    change to red steel
>
> File Changes
>
> (11 files <https://github.com/Gaelmare/WaterFlasks/pull/30/files>)
>
>    - *M* src/main/java/org/labellum/mc/waterflasks/item/FlaskItem.java
>    <https://github.com/Gaelmare/WaterFlasks/pull/30/files#diff-37a4cf538b285a5777e18f4e4b96e0495ec9da0820b3932f100953e20a5497f7>
>    (18)
>    - *M* src/main/java/org/labellum/mc/waterflasks/setup/Registration.java
>    <https://github.com/Gaelmare/WaterFlasks/pull/30/files#diff-7dcf78630999abdb7172cdee81992e30a5a151a42efa91a0135f5507b6f0c287>
>    (15)
>    - *M* src/main/resources/assets/waterflasks/lang/en_us.json
>    <https://github.com/Gaelmare/WaterFlasks/pull/30/files#diff-97bb79e846d6d4bb51184ef79cf3f48d31671d2a2fc4827c2179890dafb68a39>
>    (7)
>    - *A*
>    src/main/resources/assets/waterflasks/models/item/red_steel_flask.json
>    <https://github.com/Gaelmare/WaterFlasks/pull/30/files#diff-ee926ea18b7955773c1f1077805e669d69f6f62f126cee464958b9fa6761a821>
>    (6)
>    - *A*
>    src/main/resources/assets/waterflasks/models/item/unfinished_red_steel_flask.json
>    <https://github.com/Gaelmare/WaterFlasks/pull/30/files#diff-257a40c88846689f39e50041490c0d348712829fcb1670ada3bf5c0cb1628e5b>
>    (6)
>    - *A*
>    src/main/resources/assets/waterflasks/textures/item/red_steel_flask.png
>    <https://github.com/Gaelmare/WaterFlasks/pull/30/files#diff-0ddf650f772f6b53a58a68d98da4e71b61c87a1e6b56875afec32e4628bb4469>
>    (0)
>    - *A*
>    src/main/resources/assets/waterflasks/textures/item/unfinished_red_steel_flask.png
>    <https://github.com/Gaelmare/WaterFlasks/pull/30/files#diff-1a2d05cdaed9211c487f9a30821916875054b084d84455c3283080f30de7d41e>
>    (0)
>    - *A*
>    src/main/resources/data/waterflasks/recipes/anvil/unfinished_red_steel_flask.json
>    <https://github.com/Gaelmare/WaterFlasks/pull/30/files#diff-84dff989fe4e73192df03ff8002a0a5dd0b9693267540674bd79491753640a1a>
>    (15)
>    - *A*
>    src/main/resources/data/waterflasks/recipes/crafting/red_steel_flask.json
>    <https://github.com/Gaelmare/WaterFlasks/pull/30/files#diff-003a0b07abe1557f6d83722a7747d645616091e30c1ccd1cff631597e828f985>
>    (34)
>    - *M* src/main/resources/data/waterflasks/tags/item/flasks.json
>    <https://github.com/Gaelmare/WaterFlasks/pull/30/files#diff-2e59592f13efa44e2dd0b7b5ec9e7adfa24e8b235a74d88ccaab8b8a9b9e8da8>
>    (3)
>    - *M* src/main/resources/data/waterflasks/tags/items/flasks.json
>    <https://github.com/Gaelmare/WaterFlasks/pull/30/files#diff-380f9e79315b38fa58306a1623392bf76f3bd868807b5c105607625ac20c775d>
>    (3)
>
> Patch Links:
>
>    - https://github.com/Gaelmare/WaterFlasks/pull/30.patch
>    - https://github.com/Gaelmare/WaterFlasks/pull/30.diff
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Gaelmare/WaterFlasks/pull/30>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AN3ODQJGHGY3OJOK4FK5D6L2SM5XTAVCNFSM6AAAAABYFO33JSVHI2DSMVQWIX3LMV43ASLTON2WKOZSHA4DSNRXHE3TSMY>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

Nwitlyck committed:
Merge branch 'Gaelmare:1.20' into titanium-water-flask

Nwitlyck committed:
Merge branch 'Gaelmare:1.20' into titanium-water-flask

Nwitlyck committed:
Requested changes
-Use python resource gen for json
-Use python resource gen for book
-Change durability bar to always show itself on flasks
-Added static for red steel flask
-Updated readme to mention red steel flask
